pgcli/main.py

```diff
@@ -64,15 +64,16 @@
 from .key_bindings import pgcli_bindings
 from .packages.formatter.sqlformatter import register_new_formatter
 from .packages.prompt_utils import confirm, confirm_destructive_query
+from .packages.parseutils import is_destructive
 from .packages.parseutils import parse_destructive_warning
 from .__init__ import __version__
 
 click.disable_unicode_literals_warning = True
 
 try:
-    from urlparse import urlparse, unquote, parse_qs
+    from urlparse import urlparse
```

Comment:
Can we remove this completely? I believe this import with try/catch was designed to accommodate Python 2.7 which we no longer support.

Reply:
@amjith : I removed this try/except (and a few other leftovers of Python 2 support) in #1399.